files/tasks/build-rpm-deps.yaml

```diff
@@ -0,0 +1,6 @@
+---
+- name: Install build RPM dependencies
```

Comment:
I would switch `RPM` with `build`

Reply:
I've copied that from packit but I'll change it.